frontend/src/components/grantaccess/ForestClientCard.vue

```diff
@@ -2,44 +2,63 @@
 import Card from 'primevue/card';
 import Tag from 'primevue/tag';
 import { IconSize } from '@/enum/IconEnum';
-import type { PropType } from 'vue';
+import { ref, watchEffect, type PropType } from 'vue';
 import type { FamForestClient } from 'fam-api/dist/model/fam-forest-client';
+import { FamForestClientStatusType } from 'fam-api';
 
 const props = defineProps({
     forestClientData: {
-        type: Object as PropType<FamForestClient>,
+        type: Object as PropType<FamForestClient[]>,
     },
 });
+
+const activeClientCheck = ref<boolean>(true);
```

Comment:
I wonder if we already check if the client number is active or not before show this card, do we still need check it again here? In the GrantAccess file, we have the "forestClientInactive" for the same purpose I believe? and if the forest client is inactive, it won't be listed in the card?

Reply:
This was implemented before the scope changed to not include inactive clients to the card. You are right, now it won't be needed anymore.